identity/webapp/src/frontend/MyAccount/ChangeEmail.tsx

```diff
@@ -37,6 +38,10 @@ export const ChangeEmail: React.FC<ChangeDetailsModalContentProps> = ({ onComple
         setError('password', { type: 'manual', message: 'Incorrect password.' });
         break;
       }
+      case RequestDeleteError.BRUTE_FORCE_BLOCKED: {
```

Comment:
We're not requesting a delete here. We don't need this enum, in fact it's causing a TypeScript error:

```
Type 'RequestDeleteError.BRUTE_FORCE_BLOCKED' is not comparable to type 'UpdateUserError | undefined'.
```

```suggestion
      case UpdateUserError.BRUTE_FORCE_BLOCKED: {
```

Reply:
Fixed.